interfaces/cython/cantera/test/test_convert.py

```diff
@@ -459,3 +459,34 @@ def test_reaction_orders(self):
         self.assertNear(R.orders.get('OH'), 0.15)
         self.assertTrue(R.allow_negative_orders)
         self.assertNear(R.orders.get('H2'), -0.25)
+
+    #Here we are testing if passing a very long string will result in a Solution
+    #object. This should result in a temp file creation in most OS's
+    def test_long_source_input(self):
+        convertMech(pjoin(self.test_data_dir, 'pdep-test.inp'),
```

Comment:
Is there a reason for running `convertMech` here instead of just loading `pdep-test.cti` from `self.test_data_dir`?